Cargo.toml

```diff
@@ -17,9 +17,10 @@ serde = "1.0"
 futures = {version = "0.3", default-features = false, features = ["std", "async-await"]}
 rand = {version = "0.8", default-features = false}
 async-std = {version = "1.11", features = ["attributes"], optional = true}
```

Comment:
should async-std get a default-features=false as well?

Reply:
That would be nice, but we get a lot of errors like
```
  --> src/portal/mod.rs:37:17
   |
37 | use async_std::{fs, io, prelude::*, sync::Mutex};
   |                 ^^  ^^  ^^^^^^^     ^^^^ could not find `sync` in `async_std`
   |                 |   |   |
   |                 |   |   could not find `prelude` in `async_std`
   |                 |   no `io` in the root
   |                 no `fs` in the root
```

ill check if we can set the explicitly